src/offline/utils.ts

```diff
@@ -25,12 +25,7 @@ import {
     computeEscrowHash,
```

Comment:
You can move the utils file to `src/utils` - we're going to get rid of the `offline` directory

Reply:
Should I also move the other files in the offline directory that utils borrows functions from? (Or would that be part of getting rid of the `offline` directory?